<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NoteWindow</class>
 <widget class="QMainWindow" name="NoteWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="NoteWindow_2">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(250, 245, 230);</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="saveButton">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>saveIcon.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="nameInput">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 87 italic 12pt &quot;Segoe UI Black&quot;;</string>
        </property>
        <property name="maxLength">
         <number>100</number>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Normal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextEdit" name="textEdit">
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Segoe UI&quot;;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
